registration/app/services/Configuration.scala

```diff
@@ -0,0 +1,32 @@
+package services
+
+import javax.inject.Inject
+
+import com.gu.conf.ConfigurationFactory
+
+import scala.concurrent.ExecutionContext
+import scalaz.\/
+import scalaz.std.option.optionSyntax._
+
+case class NotificationHubConfiguration(
+  connectionString: String,
+  hubName: String
+)
+
+case class ErrorMessage(message: String)
+
+final class Configuration @Inject()()(implicit executionContext: ExecutionContext) {
+
+  private lazy val conf = ConfigurationFactory.getConfiguration(
+    applicationName = "notification",
```

Comment:
"registration" ? or are we sharing the same config file (which is actually not a bad idea)